googletest/feature_support_test.cpp

```diff
@@ -2,7 +2,9 @@
 // Licensed under the MIT License.
 #include "gtest/gtest.h"
 
-#ifndef _WIN32
+#ifdef _WIN32
+#include <unknwn.h>
```

Comment:
Needs a tweak because it's ``unkwn.h`` in the MinGW headers, but it's ``Unknwn.h`` in the Windows SDK so Clang will complain with a ``-Wnonportable-system-include-path``